server/services/party/party.controller.ts

```diff
@@ -1,20 +1,26 @@
-import { Controller } from '@nestjs/common'
-import { PartyService } from './party.service'
-import { MessagePattern } from '@nestjs/microservices'
-import { WORLD_PREFIX } from '../world/world.prefix'
-import { GetParty, CreateParty } from './actions'
+import { Controller }                           from '@nestjs/common'
+import { PartyService }                         from './party.service'
+import { MessagePattern }                       from '@nestjs/microservices'
+import { WORLD_PREFIX }                         from '../world/world.prefix'
+import {GetParty, CreateParty, MakePartyLeader} from './actions'
 
 @Controller()
 export class PartyController {
     constructor(private service: PartyService) {}
 
     @MessagePattern(WORLD_PREFIX + GetParty.event)
     getParty(data: GetParty) {
-        return this.service.getPartyName(data.partyId)
+        return this.service.getPartyByLeader(data.leaderId)
```

Comment:
Most UIs on the client may not even care about partyId, so being able to reference the party by the leader's character ID would be a bit more flexible.